frontend/packages/process-editor/test/EventListeners.ts

```
@@ -0,0 +1,58 @@
+import { ArrayUtils } from '@studio/pure-functions';
+
+export class EventListeners {
+  private list: Map<string, Function[]>;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)